tests/integration/utils.py

```diff
@@ -201,6 +208,12 @@ def update_test_data_if_needed(path: Path, data: Dict) -> None:
             file.write(json_formatted_str + "\n")
 
 
+def _fetch_cyclone_dx_schema() -> dict[str, Any]:
+    response = requests.get(CYCLONEDX_SCHEMA_URL)
+    response.raise_for_status()
+    return response.json()
```

Comment:
We could `@functools.cache` this to avoid re-fetching the data in every test

Reply:
Added :+1: